de-lib/src/main/java/org/iplantc/de/client/models/tool/ToolAutoBeanFactory.java

```diff
@@ -33,4 +34,20 @@
 
     AutoBean<SimpleServiceError> simpleServiceError();
 
+    /**
+     * Build a tool with default values
+     *
+     * @return  a tool
+     */
+    default AutoBean<Tool> geTool() {
+        Tool tool = getTool().as();
```

Comment:
Actually the method name is `geTool()`, note the missing `t` and that why java didn't complain. I went ahead and renamed the method name.